src/index.ts

```diff
@@ -86,6 +86,7 @@ class FlakeCheckerAction {
         ...executionEnv,
         ...process.env, // To get $PATH, etc
       },
+      ignoreReturnCode: true,
```

Comment:
This signals to the exec library to not raise an exception on a non-zero error. Since we handle that case immediately below, we don't need it.